modules/v2x/proto/v2x_obu_rsi.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto2";
+
+package apollo.v2x.obu.rsi;
```

Comment:
What about `package apollo.v2x.obu`.

What I mean is, all proto files `v2x_xxx` goes to package `apollo.v2x` and all proto files with name `v2x_obu_` goes to package `apollo.v2x.obu`.